src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -0,0 +1,155 @@
+/* global Promise */
+import { _ } from 'okta';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals/BaseFooter';
+
+const request = (url, method = 'GET', body = null, timeout = 1000) => {
```

Comment:
why not use jquery.ajax or  'cross-fetch';

Reply:
I thought we try to get rid of jquery?